src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -369,8 +348,9 @@ non-supporting browsers.
 ## Demo
 
 If you are anything like me, you keep a close eye on the development of the various
-cryptocurrencies. (In reality I very much do not, but, for the sake of this article, just assume I
-do.) To keep track of the cryptocurrencies that I own, I have developed a web app that allows me to
+cryptocurrencies. (In reality I very much do not because I love this planet, but,
+for the sake of this article, just assume I
+did.) To keep track of the cryptocurrencies that I own, I have developed a web app that allows me to
```

Comment:
Or you could just do your cryptomining on your secret moonbase.

Reply:
🚀🌕

I rather try preserve the planet we have.